MAUI/DataGrid/pull-to-refresh.md

```diff
@@ -10,6 +10,135 @@ keywords : maui datagrid, maui grid, grid maui, maui gridview, grid in maui, .ne
+
+To enable and perform the pull-to-refresh operation, follow the code example:
+
+{% tabs %}
+
+{% highlight Xaml %}
+
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                       AllowPullToRefresh="True"
+                       PullToRefreshCommand="{Binding RefreshCommand}"
+                       ItemsSource="{Binding OrdersInfo}">
+{% endhighlight %}
+
+{% highlight c# %}
+
+dataGrid.AllowPullToRefresh = true;
+
+Command RefreshCommand = new Command(ExecutePullToRefreshCommand);
+dataGrid.PullToRefreshCommand = RefreshCommand;
+ 
+private async void ExecutePullToRefreshCommand()
+{
+    this.dataGrid.IsBusy = true;
+    await Task.Delay(new TimeSpan(0, 0, 5));
+    viewModel.ItemsSourceRefresh();
+    this.dataGrid.IsBusy = false;
+}
+
+//ViewModel.cs
+public void ItemsSourceRefresh()
+{
+    int count = random.Next (1, 10);
+    for (int i = 1; i <= count; i++) 
+    {
+        this.OrdersInfo!.Insert(0, new OrderInfo()
+        {
+            OrderID = i,
+            CustomerID = this.customerID[this.random.Next(15)],
+            EmployeeID = this.random.Next(1700, 1800),
+        });
+    }        
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Pull To Refresh" src="Images\pull-to-refresh\net-maui-pull-to-refresh-slide-on-top.gif" width="404"/>
+
+### Transition Mode
+
+The [TransitionMode]() property specifies the mode of the animations. It has two modes:
+
+* [SlideOnTop]()
+* [Push]()
+
+The default transition is `SlideOnTop`, which draws the RefreshView on top of the `DataGrid`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                       TransitionType="SlideOnTop"
+                       AllowPullToRefresh="True"
+                       ItemsSource="{Binding OrdersInfo}">
+
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="TransitionMode SlideOnTop" src="Images\pull-to-refresh\net-maui-pull-to-refresh-slide-on-top.gif" width="404"/>
+
+The following code example demonstrates how to set the `TransitionMode` to `Push` in SfPullToRefresh. This transition moves only the refresh content, while the header remains fixed.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                       TransitionType="Push"
+                       AllowPullToRefresh="True"
+                       ItemsSource="{Binding OrdersInfo}">
+
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="TransitionMode Push" src="Images\pull-to-refresh\net-maui-pull-to-refresh-push.gif" width="404"/>
+
+### Customization
+
+The data grid allows you to customize various properties such as TransitionMode, PullingThreshold, ProgressBackground, ProgressColor, and more. Refer to the code example below to customize the ProgressColor and ProgressBackground of the pull-to-refresh view.
```

Comment:
You can customize various properties of `SfPullToRefresh` in the data grid, including `TransitionMode`, `PullingThreshold`, `ProgressBackground`, `ProgressColor`, and more.

Reply:
modified